custom-metrics-stackdriver-adapter/pkg/adapter/translator/utils/filter_builder.go

```diff
@@ -192,11 +194,18 @@ func (fb *FilterBuilder) WithNamespace(namespace string) *FilterBuilder {
 //  2. To add "resource.lables.pod_name = one_of(my-pod-1,my-pod-2)"
 //     filterBuilder := NewFilterBuilder(SchemaTypes[PodSchemaKey]).WithPods([]string{"my-pod-1", "my-pod-2"})
 func (fb *FilterBuilder) WithPods(pods []string) *FilterBuilder {
```

Comment:
Is there a reason to have a pointer return type here? With the builder pattern it should not be needed afaik.

Reply:
gotcha, that's a good point, will update other places as well.